app/src/main/kotlin/com/simplemobiletools/smsmessenger/extensions/String.kt

```diff
@@ -0,0 +1,16 @@
+package com.simplemobiletools.smsmessenger.extensions
+
+fun String.getExtensionFromMimeType(): String {
+    return when (this) {
```

Comment:
make sure you convert the extension to lowercase before checking, just to be sure